sonar-xml-plugin/src/test/resources/checks/MimeNosniffCheck/webconfig-compliant/web.config

```
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <system.webServer>
+    <httpProtocol>
+      <customHeaders>
+        <add name="X-Content-Type-Options" value="nosniff"/>
```

Let's add this. It would be typical thing to do. And if we ever fix the FN from the previous comment, it will help uncover lost TN
```suggestion
        <clear />
        <add name="X-Content-Type-Options" value="nosniff"/>
```